sympy/geometry/ellipse.py

```diff
@@ -1557,7 +1557,7 @@ def __new__(cls, *args, **kwargs):
             y = kwargs.get('y', 'y')
             equation = args[0].expand()
             if isinstance(equation, Eq):
-                equation = equation.lhs - equation.rhs
+                equation = equation.lhs_rhs()
```

Comment:
```suggestion
               equation = (_:= equation).lhs - _.rhs
```

Reply:
I don't see how
```
equation = (_:= equation).lhs - _.rhs
```
is better than the original:
```
equation = equation.lhs - equation.rhs
```

Reply:
I was interrupted mid-edit